setup.py

```diff
@@ -15,7 +15,7 @@
 # This call to setup() does all the work
 setup(
     name="promoted-python-delivery-client",
-    version="0.1.0",
+    version="0.0.0",
```

Comment:
Interesting.  Why?  Because we're pre-release?

Reply:
Because when I next run the release process it will increment.